<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalogImport_Widget</class>
 <widget class="QWidget" name="AnalogImport_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analog Time Series Import</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <item>
    <layout class="QHBoxLayout" name="dataNameLayout">
     <item>
      <widget class="QLineEdit" name="data_name_text">
       <property name="text">
        <string>analog_1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_data_name">
       <property name="text">
        <string>Data Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loaderTypeLayout">
     <item>
      <widget class="QLabel" name="label_loader_type">
       <property name="text">
        <string>Format:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="loader_type_combo">
       <item>
        <property name="text">
         <string>CSV</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Binary</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="loaderTypeSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stacked_loader_options">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="CSVAnalogImport_Widget" name="csv_analog_import_widget"/>
     <widget class="BinaryAnalogImport_Widget" name="binary_analog_import_widget"/>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CSVAnalogImport_Widget</class>
   <extends>QWidget</extends>
   <header>AnalogTimeSeries/CSV/CSVAnalogImport_Widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BinaryAnalogImport_Widget</class>
   <extends>QWidget</extends>
   <header>AnalogTimeSeries/Binary/BinaryAnalogImport_Widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
